.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
this file shouldn't have been commited

Reply:
I’ll remove it and create a PR again. Hope that would help and also I want to ask if the function works as requested?